pages/generative-apis/troubleshooting/fixing-common-issues.mdx

```diff
@@ -31,14 +31,24 @@ Below are common issues that you may encounter when using Generative APIs, their
 
 ### Cause
 - You performed too many API requests over a given minute
-- You consumed too many tokens (input and output) with your API requests over a given minute 
+- You consumed too many tokens (input and output) with your API requests over a given minute
 
 ### Solution
-- [Ask our support](https://console.scaleway.com/support/tickets/create) to raise your quota
-- Smooth out your API requests rate by limiting the number of API requests you perform in parallel
-- Reduce the size of the input or output tokens processed by your API requests
+- Smooth out your API requests rate by limiting the number of API requests you perform over a given minute so that you remain below your [organization quotas for Generative APIs](https://www.scaleway.com/en/docs/organizations-and-projects/additional-content/organization-quotas/#generative-apis).
+- [Add a payment method](https://www.scaleway.com/en/docs/billing/how-to/add-payment-method/#how-to-add-a-credit-card) and [validate your identity](https://www.scaleway.com/en/docs/account/how-to/verify-identity/) to increase automatically your quotas [based on standard limits](https://www.scaleway.com/en/docs/organizations-and-projects/additional-content/organization-quotas/#generative-apis).
+- [Ask our support](https://console.scaleway.com/support/tickets/create) to raise your quota.
+- Reduce the size of the input or output tokens processed by your API requests.
 - Use [Managed Inference](/managed-inference/), where these quota do not apply (your throughput will be only limited by the amount of Inference Deployment your provision)
 
+## 429: Too Many Requests - You exceeded your current threshold of concurrent requests
+
+### Cause
+- You kept too many API requests opened at the same time (number of HTTP sessions opened in parallel)
+
+### Solution
+- Smooth out your API requests rate by limiting the number of API requests you perform at the same time (eg. requests which did not receive a complete response and are still opened) so that you remain below your [organization quotas for Generative APIs](https://www.scaleway.com/en/docs/organizations-and-projects/additional-content/organization-quotas/#generative-apis).
```

Comment:
```suggestion
- Smooth out your API requests rate by limiting the number of API requests you perform at the same time (eg. requests which did not receive a complete response and are still opened) so that you remain below your [organization quotas for Generative APIs](/organizations-and-projects/additional-content/organization-quotas/#generative-apis).
```